testsuite/forge.py

```diff
@@ -401,7 +401,7 @@ def get_humio_forge_link(
     urlparts = {
         "query": (
             "$forgeLogs(validator_instance=*)"
-            f" | {forge_namespace}"
+            f' | "k8s.namespace" = "{forge_namespace}"'
             ' | "k8s.labels.app.kubernetes.io/name" = forge'
```

Comment:
I don't see this (k8s.labels.app.kubernetes.io/name) in the query? And adding it, shows no results?

Reply:
Actually that's a great catch. Looks like we're generating the links separately in two places. I've updated both of them now.